### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by danilodileo**
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,20 +16,38 @@
...
 
 <!-- TODO nf-core: Add full-sized test dataset and amend the paragraph below if applicable -->
-On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/metatdenovo/results).
+On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/metatdenovo/results) e.g. the results for running the pipeline with --assembler will be in a folder called orf_caller and so on.
```
Comment (emnilsson) [inline README.md]: The last bit, 

> e.g. the results for running the pipeline with --assembler will be in a folder called orf_caller and so on.

doesn't really make sense to me. Why would assembly results end up in an orf_caller-folder? Shouldn't it end up in an assembly-folder?

File: README.md [hunk 2]
```diff
@@ -16,20 +16,38 @@
...
 
 ## Pipeline summary
 
-<!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
-
+![nf-core/rnaseq metro map](docs/images/nf-core-metatdenovo_metro_map.png)
 1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
 2. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
+3. Quality trimming and adapters removal for raw reads ( [`Trimm Galore!`](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/))
+4. Merge pair-end for cleaned reads ( [`Seqtk`](https://github.com/lh3/seqtk))
```
Comment (emnilsson) [inline README.md]: Maybe: "Merge cleaned, pair-end reads" or maybe: "Merge trimmed, pair-end reads"?

File: README.md [hunk 3]
```diff
@@ -16,20 +16,38 @@
...
+5. Choice of de novo assembly program route:
+    1. [`RNAspade`](https://cab.spbu.ru/software/rnaspades/) Suggested for Eukaryotes de novo assembly
+    2. [`Megahit`](https://github.com/voutcn/megahit) Suggested for Prokaryotes de novo assembly
```
Comment (emnilsson) [inline README.md]: Maybe non-capital letters on the start of "Suggested"? Or capital letters on "suggested" in the subpoints under step 6 and also on step 7 point 1?

File: README.md [hunk 4]
```diff
@@ -16,20 +16,38 @@
...
+6. Orf caller step:
```
Comment (emnilsson) [inline README.md]: Maybe: "Choice of orf caller", to be more consistent with point 5 (also, maybe remove "program route" for that one?).

File: README.md [hunk 5]
```diff
@@ -16,20 +16,38 @@
...
+    1. [`TransDecoder`](https://github.com/TransDecoder/TransDecoder) suggested for Eukaryotes
+    2. [`Prokka`](https://github.com/tseemann/prokka) suggested for Prokaryotes
+    3. [`Prodigal`](https://github.com/hyattpd/Prodigal) suggested for Prokaryotes
+7. quantification route:
```
Comment (emnilsson) [inline README.md]: Capital letter in "quantification", and maybe a bit more info? "Quantification of genes identified in assemblies" or something?

File: README.md [hunk 6]
```diff
@@ -16,20 +16,38 @@
...
+    1. generate index of assembly [`BBmap index`](https://sourceforge.net/projects/bbmap/) 
+    2. Mapping cleaned reads to the assembly for quantification [`BBmap`](https://sourceforge.net/projects/bbmap/) 
+    3. Get raw counts per orf samples [`Featurecounts`](http://subread.sourceforge.net) -> TSV table with collected featurecounts output
```
Comment (emnilsson) [inline README.md]: What does "per orf samples" mean?

File: README.md [hunk 7]
```diff
@@ -16,20 +16,38 @@
...
+    2. [`Megahit`](https://github.com/voutcn/megahit) Suggested for Prokaryotes de novo assembly
+6. Orf caller step:
+    1. [`TransDecoder`](https://github.com/TransDecoder/TransDecoder) suggested for Eukaryotes
+    2. [`Prokka`](https://github.com/tseemann/prokka) suggested for Prokaryotes
+    3. [`Prodigal`](https://github.com/hyattpd/Prodigal) suggested for Prokaryotes
+7. quantification route:
+    1. generate index of assembly [`BBmap index`](https://sourceforge.net/projects/bbmap/) 
+    2. Mapping cleaned reads to the assembly for quantification [`BBmap`](https://sourceforge.net/projects/bbmap/) 
+    3. Get raw counts per orf samples [`Featurecounts`](http://subread.sourceforge.net) -> TSV table with collected featurecounts output 
+8. taxonomical annotation route:
```
Comment (emnilsson) [inline README.md]: Maybe just "Taxonomical annotation"? (With capital "T".)

File: README.md [hunk 8]
```diff
@@ -16,20 +16,38 @@
...
+    1. [`EUKulele`](https://github.com/AlexanderLabWHOI/EUKulele) -> Reformat TSV output "Reformat_tax.R"
+9. Functional annotation routes:
```
Comment (emnilsson) [inline README.md]: Maybe "Choice of functional annotation route/s"? For consistency..

**Review by emnilsson — APPROVED** [review]
LGTM but a lot of tiny stuff, mostly taste-specific, do what you like with it.

File: README.md (hunk 1 above)
Comment (danilodileo) [reply README.md]: yes. I changed in: "the results for running the pipeline with --assembler will be in a folder called assembly and so on"

File: README.md (hunk 2 above)
Comment (danilodileo) [reply README.md]: changed with "Merge trimmed, pair-end reads"

File: README.md (hunk 3 above)
Comment (danilodileo) [reply README.md]: I changed with non-capital letters

File: README.md (hunk 6 above)
Comment (danilodileo) [reply README.md]: It should be more "per each gene present in the assembly" or something similar

**Pull request merged**